R/onehot_encoder.R

```diff
@@ -0,0 +1,22 @@
+#' onehot_encoder
+#'
+#' This function encodes categorical variables using the popular onehot method for each category.
+#'
+#' @param X_train A tibble representing the training data set containing some categorical features/columns.
+#' @param X_test A tibble representing the test set, containing some set of categorical features/columns.
+#' @param cat_columns A character vector containing the names of the categorical columns in the tibble or
+#' data frame that should be encoded.
+#'
+#' @return A list containing with processed training and test sets, in which the named categorical
+#' columns are replaced with their encodings.
```

Comment:
There's a typo here. Should just say "A list with processed training". No containing needed.